.github/workflows/publish.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: conda init bash && exec $SHELL && conda activate base
+      - run: conda run pip install --upgrade build pip setuptools wheel
+      - run: conda run pip install --editable ".[dev, docs]"
+      - run: conda run python -m build
+      - uses: pypa/gh-action-pypi-publish@v1.4.2
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
+          skip_existing: true
+          user: __token__
+on:
+  pull_request:
```

Comment:
I don't think we want this trigger. You want to publish to PyPI every PR?

Reply:
Using this to test.